helm/efk-stack-app/charts/opendistro-es/templates/kibana/kibana-deployment.yaml

```diff
@@ -83,6 +83,10 @@ spec:
       {{- end }}
 {{- if .Values.kibana.extraEnvs }}
 {{ toYaml .Values.kibana.extraEnvs | indent 8 }}
+{{- end }}
+{{- if .Values.kibana.command }}
+        command: 
+{{ toYaml .Values.kibana.command | indent 8 }}
```

Comment:
Lacking knowledge here, please help out :) What happens when this is used in values?
```
command: ["/usr/local/bin/kibana-docker"]
```
Will it work?

Reply:
After running helm template, the output is:
        command: 
        - /usr/local/bin/kibana-docker